salt/_states/metalk8s.py

```diff
@@ -1,28 +1,39 @@
 """Custom states for MetalK8s."""
 
+import logging
+import tempfile
```

Comment:
Is this used? Seems surprising in a state module.

Reply:
No indeed, good catch.